### ISSUE TITLE
Affine Transform

### ISSUE THREAD

**Issue by twkim1986** [MVIG-SJTU/AlphaPose#897]
What is the criteria for selecting 3 points for affine transform?

https://github.com/MVIG-SJTU/AlphaPose/blob/41642dbc915abdc4772cf773023938c9bcf3ab9f/alphapose/utils/transforms.py#L761

**Comment by thamnt2611** [comment]
No, you only need to select 3 non-linear points. https://docs.opencv.org/4.x/d4/d61/tutorial_warp_affine.html
